lib/ReactViews/Analytics/GenericParameterEditor.tsx

```diff
@@ -16,12 +16,13 @@ export default class GenericParameterEditor extends React.Component<{
   }
 
   render() {
+    const value = (this.props.parameter.value || "") as string;
     return (
```

Comment:
Ensure `value` is never `undefined` to prevent React warning about [uncontrolled input](https://reactjs.org/docs/uncontrolled-components.html).